internal/k8s/handlers.go

```diff
@@ -154,6 +154,7 @@ func createSecretHandlers(lbc *LoadBalancerController) cache.ResourceEventHandle
 		AddFunc: func(obj interface{}) {
 			secret := obj.(*v1.Secret)
 			if err := lbc.ValidateSecret(secret); err != nil {
+				glog.V(3).Infof("Error validating secret %v: %v", secret.Name, err)
```

Comment:
This error is not always an error (and in line 173) Consider other secrets in the cluster not used by the Ingress Controller.  For every secret like that, the error debug message will be printed. Will it be ok? For debug purposes, perhaps it makes sense to change the message, to something like `Ignoring secret of unknown kind`

I think the problem her is that we should have had a different function here in the first place. Something like `IsIngressControllerSecret` or `HasKnownSecretKind`, where it would return true/false if the Ingress Controller was interested in that secret. (But the name is not needed to be addressed in this PR though)